spec/spec_helper.rb

```diff
@@ -1,14 +1,11 @@
 # Make sure specs run with the definitions from test.rb
 ENV['ROBOT_ENVIRONMENT'] = 'test'
 
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
+require 'simplecov'
+SimpleCov.start
+
 require 'coveralls'
 Coveralls.wear!
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
-  CodeClimate::TestReporter::Formatter,
-  Coveralls::SimpleCov::Formatter
-]
```

Comment:
Ok, Stacks is doing just the `SimpleCov.start` way now. See https://github.com/sul-dlss/stacks/commit/0c9a534548f44f83fb1602ba251303a37507b3a4